Magento_Swatches/templates/product/layered/renderer.phtml

```diff
@@ -28,61 +30,67 @@
                     $swatchData['swatches'][$option]['value']); ?>
                     <?php $swatchImagePath = $block->getSwatchPath('swatch_image',
                     $swatchData['swatches'][$option]['value']); ?>
-                    <a
-                        href="<?= $label['link'] ?>"
-                        aria-label="<?= $label['label'] ?>"
-                    >
-                        <div
-                            tabindex="-1"
-                            option-type="2"
-                            option-id="<?= $option ?>"
-                            option-label="<?= $label['label'] ?>"
-                            option-tooltip-thumb="<?= $swatchThumbPath ?>"
-                            style="background: url(<?= $swatchImagePath ?>) no-repeat center; background-size: cover;"
-                        ></div>
-                    </a>
+                    <li>
+                        <a
+                            href="<?= $label['link'] ?>"
+                            aria-label="<?= $label['label'] ?>"
+                        >
+                            <div
+                                tabindex="-1"
+                                option-type="2"
+                                option-id="<?= $option ?>"
+                                option-label="<?= $label['label'] ?>"
+                                option-tooltip-thumb="<?= $swatchThumbPath ?>"
+                                style="background: url(<?= $swatchImagePath ?>) no-repeat center; background-size: cover;"
+                            ></div>
+                        </a>
+                    </li>
                 <?php
                 break;
                 case '1':
                 ?>
-                    <a
-                        href="<?= $label['link'] ?>"
-                        aria-label="<?= $label['label'] ?>"
-                        class="filter__color-option"
-                    >
-                        <div
-                            class="filter__color-sample"
-                            tabindex="-1"
-                            option-type="1"
-                            option-id="<?= $option ?>"
-                            option-label="<?= $label['label'] ?>"
-                            option-tooltip-value="<?= $swatchData['swatches'][$option]['value'] ?>"
-                            style="background: <?= $swatchData['swatches'][$option]['value'] ?>"
-                        ></div>
-                        <span class="filter__color-label">
-                            <?= $label['label'] ?>
-                        </span>
-                    </a>
+                    <li class="filter__color-item">
+                        <a
+                            href="<?= $label['link'] ?>"
+                            aria-label="<?= $label['label'] ?>"
```

Comment:
if the label is inside the `span.filter__color-label` it shouldn't be necessary. Unless the label is sometimes missing, so we can leave aria-label then.

Reply:
removed

Reply:
👍